BXMSDOC-4588-master: Update interacting with processes and tasks doc per changes in 7.5

kaldesai opened the pull request:
- [Epic](https://issues.jboss.org/browse/BXMSDOC-4412)
- [JIRA](https://issues.jboss.org/browse/BXMSDOC-4588)
- [Interacting with Processes & tasks 7.5 RHPAM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4588-IPT-RHPAM/#interacting-with-processes-bulk-actions-on-tasks-con)

Hey @nmirasch and @jstastny-cz, 
I have restructured the content and created a new section **2.6. Bulk actions on tasks** in the document and created the subsections for each bulk actions. 

Can you please review and verify the new section **2.6. Bulk actions on tasks** that I have added in the interacting with Processes & tasks document? I have provided the RHPAM doc preview link and the respected JIRA.
Please take a look and let me know what do you think. 

Thank you.

jstastny-cz commented on doc-content/enterprise-only/processes/interacting-with-processes-performing-bulk-actions-proc.adoc:
```diff
@@ -0,0 +1,17 @@
+* *Bulk Release*: To release the tasks in bulk, select two or more tasks. From the *Bulk Actions* drop-down, select *Bulk Release* and click *Release* on the *Release selected tasks* window.
+* *Bulk Resume*: To resume the tasks in bulk, select two or more tasks from the *Task* column. From the *Bulk Actions* drop-down, select *Bulk Resume* and click *Resume* on the *Resume selected tasks* window.
+* *Bulk Suspend*: To suspend the tasks in bulk, select two or more tasks from the *Task* column. From the *Bulk Actions* drop-down, select *Bulk Suspend* and click *Suspend* on the *Suspend selected tasks* window.
+* *Bulk Reassign*: To reassign the tasks in bulk, select two or more tasks from the *Task* column. From the *Bulk Actions* drop-down, select *Bulk Reassign*. In the *Tasks reassignment* window, enter the user identifier of the user to reassign the tasks, and click *Delegate*.
```
1. Unify "To claim the tasks in bulk, select two or more tasks." vs. "To reassign the tasks in bulk, select two or more tasks from the *Task* column."
2. select two or more tasks from the *Task* column. -> ...... from the *Task* table.
3. Can we add more structure? It is supposed to be a procedure, but rather is a list of operations. It could help if for each operation there was a list of steps instead of the long description. So we could end up with nested list. wdyt?

kaldesai replied:
Hey, I will restructure it in procedural manner.

jstastny-cz commented on doc-content/enterprise-only/processes/interacting-with-processes-performing-bulk-actions-proc.adoc:
```diff
@@ -0,0 +1,17 @@
++
+For each task selected, a notification is displayed showing the reassignment result.
+
+If a specified bulk action is not permitted based on the task status, a notification is displayed and the operation is not executed on tasks.
```
"is not executed on tasks" that's not correct. The operation is not executed only on the non-suitable tasks, the others are executed just fine. .... so it might help to change:
is not executed on tasks -> is not executed on that particular task
or sth in this manner.

nmirasch replied:
+1

jstastny-cz commented on doc-content/enterprise-only/processes/interacting-with-processes-performing-bulk-actions-proc.adoc:
```diff
@@ -0,0 +1,17 @@
+* *Bulk Suspend*: To suspend the tasks in bulk, select two or more tasks from the *Task* column. From the *Bulk Actions* drop-down, select *Bulk Suspend* and click *Suspend* on the *Suspend selected tasks* window.
+* *Bulk Reassign*: To reassign the tasks in bulk, select two or more tasks from the *Task* column. From the *Bulk Actions* drop-down, select *Bulk Reassign*. In the *Tasks reassignment* window, enter the user identifier of the user to reassign the tasks, and click *Delegate*.
++
+For each task selected, a notification is displayed showing the reassignment result.
```
move this one level up (applicable to all operations) and replace 'reassignment' with 'operation'

nmirasch replied:
+1

kaldesai replied:
Noted.

jstastny-cz commented on doc-content/enterprise-only/processes/interacting-with-processes-bulk-actions-on-tasks-con.adoc:
```diff
@@ -0,0 +1,6 @@
+
+In the *Tasks* and *Task Inbox* pages in {CENTRAL}, you can perform bulk actions over multiple tasks in a single operation.
+
+NOTE: If a specified bulk action is not permitted based on the task status, a notification is displayed and the operation is not executed on that particular tasks. 
```
on that particular tasks. -> on that particular task.

nmirasch replied:
@kaldesai +1 to 'that particular task.'

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-claiming-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+[id='interacting-with-processes-claiming-tasks-in-bulk-proc']
+= Claiming tasks in bulk
+
+After tasks are created in {CENTRAL}, you can claim the released tasks in bulk.
```
@kaldesai I think it's better talk about available tasks instead of released tasks.

kaldesai replied:
@nmirasch, Ack

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-claiming-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+. From the *Bulk Actions* drop-down list, select *Bulk Claim*.
+. Click *Claim* on the *Claim selected tasks* window to confirm.
+
+For each task selected, a notification is displayed showing the updated result.
```
@kaldesai here you said that the notification show the 'updated result', I would say that the notification shows the action or operation result.

kaldesai replied:
@nmirasch, Noted :)

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-reassigning-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,13 @@
+. In the *Tasks reassignment* window, enter the user identifier of the user to reassign the tasks.
+. Click *Delegate*.
+
+For each task selected, a notification is displayed showing the updated result.
```
@kaldesai same here :)

kaldesai replied:
@nmirasch, Ack :)

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-releasing-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+[id='interacting-with-processes-releasing-tasks-in-bulk-proc']
+= Releasing tasks in bulk
+
+After tasks are created in {CENTRAL}, you can release your tasks in bulk for others to claim.
```
@kaldesai it's not enough that task are created, you need they are assigned to you. I would remove the first part: After tasks are created in {CENTRAL}. Maybe introduce 'your owned tasks' here would be clearer that you are the owner

kaldesai replied:
Hey @nmirasch, 
wdyt?
I will just add the following sentence..
"_You can release your owned tasks in bulk for others to claim._"

nmirasch replied:
looks good!

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-releasing-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+. From the *Bulk Actions* drop-down list, select *Bulk Release*.
+. Click *Release* on the *Release selected tasks* window to confirm.
+
+For each task selected, a notification is displayed showing the updated result.
```
@kaldesai Maybe just removing 'updated' could work

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-resuming-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+[id='interacting-with-processes-resuming-tasks-in-bulk-proc']
+= Resuming tasks in bulk
+
+After tasks are created in {CENTRAL}, you can resume your tasks in bulk.
```
What you can do is resume suspended tasks available to you, it's not enough that task are created in {CENTRAL}
Maybe something like this 'After task are suspended in {CENTRAL} they can be resumed in bulk'

kaldesai replied:
@nmirasch, Ack

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-resuming-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+. From the *Bulk Actions* drop-down list, select *Bulk Resume*.
+. Click *Resume* on the *Resume selected tasks* window to confirm.
+
+For each task selected, a notification is displayed showing the updated result.
```
@kaldesai remove 'updated'?

kaldesai replied:
@nmirasch, Noted

nmirasch commented on doc-content/enterprise-only/processes/interacting-with-processes-suspending-tasks-in-bulk-proc.adoc:
```diff
@@ -0,0 +1,12 @@
+. From the *Bulk Actions* drop-down list, select *Bulk Suspend*.
+. Click *Suspend* on the *Suspend selected tasks* window to confirm.
+
+For each task selected, a notification is displayed showing the updated result.
```
@kaldesai remove 'updated'?

kaldesai replied:
@nmirasch, Noted :)

kaldesai commented:
Hey @nmirasch and @jstastny-cz,
I have incorporated the suggested changes in the document. 
Can you please review and verify the doc once again? I have updated the rendered output link.

kaldesai commented:
Hey @emmurphy1, I have incorporated all your suggestions in the doc.
